cmd/thanos/rule.go

```diff
@@ -295,10 +295,10 @@ func runRule(
 			return err
 		}
 	} else {
-		for _, addr := range queryAddrs {
-			if addr == "" {
-				return errors.New("static querier address cannot be empty")
-			}
+		var err error
```

Comment:
let's change returned `) error {` to `) (err error) {` so you don't need this and on line 292

Reply:
I have move `var err error` next to definition of `var queryCfg []query.Config`.